src/lib/VirtualScroller.js

```diff
@@ -123,6 +120,23 @@ class VirtualScroller {
         }
     }
 
+    /**
+     * Given the container height, calculate the virtual window properties
+     * @param {number} containerHeight - the available container height of the virtual scroller
```

Comment:
due to the fact that getContainerHeight may return null I think we should also expect null for the param here.

Reply:
Seeing as null would be an invalid input and the method would fail fast, does it make sense to document null as a valid param?